### PR TITLE
Home page updates

### PR DESCRIPTION
**Pull request by kildre**
Updates to the home page, mainly stylistic changes, but also text updates and iconography.

## Reference Issues or PRs
https://gitlab.jatic.net/jatic/team-metrostar/t-e-platform/-/issues/595

This update addresses the issues in the attached ticket.
## What does this implement/fix?
This implementation fixes stylistic and design issues
_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [ ] Documentation Update
- [x] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [x] Did you add new tests?

#### Text styling

- [ ] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [ ] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [ ] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [ ] This content adheres to the Nebari style guides.

#### Non-text content

- [ ] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [ ] If there are emojis, there are not more than three in a row.
- [ ] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [ ] If the content were to be read as plain text, it still makes sense, and no information is missing.

### CODE REVIEW

File: ui/src/components/app-card/app-card.tsx [hunk 1]
```diff
@@ -207,7 +207,7 @@ export const AppCard = ({
             ) : (
               <></>
             )}
-            <CardMedia>
+            {/* <CardMedia>
```
Comment (jbouder) [inline ui/src/components/app-card/app-card.tsx:210]: Can this removed?

**Review by jbouder — CHANGES_REQUESTED** [review]
Code-wise looks great, just 1 thing to cleanup. Also, please include the following in this PR:
1. Fix the header opacity
2. Fix the row spacing between App rows
3. Ensure the home navigation button can be themed

File: ui/src/components/app-card/app-card.tsx (hunk 1 above)
Comment (kildre) [reply ui/src/components/app-card/app-card.tsx]: Removed

**Comment by kildre** [comment]
> Code-wise looks great, just 1 thing to cleanup. Also, please include the following in this PR:
> 
> 1. Fix the header opacity
> 2. Fix the row spacing between App rows
> 3. Ensure the home navigation button can be themed

√, √, √

**Review by jbouder — APPROVED** [review]
Looks great!

**Pull request merged**